README.md

```diff
@@ -277,28 +278,160 @@ This gives the start timestamps that can be used to correlate the events and the
 
 ## Unit tests
 
-Unit tests can be run with `go test` outside of Docker if you have the correct C++ dependencies installed on your system.
+ 0chain unit tests verify the behaviour of individual parts of the program. A config for the base docker image can be provided on run to execute general unit tests.
+
+
+![unit testing uml](https://user-images.githubusercontent.com/65766301/120052862-0b4ffd00-c045-11eb-83c8-977dfdb3038e.png)
+
+### Getting started
+
+### Prerequisites
+
+Docker and Git must be installed in order to run the tests .
```

Comment:
Docker and Git must be installed to run the tests. `in order to` can be shortened to `to`. Plus extra space before the terminating period.